3rdparty/libunwind/CMakeLists.txt

```diff
@@ -17,6 +17,24 @@ else()
   )
 endif()
 
+if (WIN32)
+  find_program(
+       BASH
+       NAMES bash
+       PATHS "C:/Program Files/Git/bin"
```

Comment:
It looks live we always expect Git to be installed in a specific path which may not always be the case.
It would be better to find Git first and then figure out the path to git bash from Git's path.

Reply:
Sounds ok. Will additionally have to not pick the path in Windows system directory only.

Reply:
@anakrish I think with Git Bash on Windows, the current logic is a safe bet.

Reply:
Agree with andschwa. Reverted to the previous logic.